File: TaskMasterAPI/src/main/java/com/devlab/taskmaster/dto/report/ReportDTO.java
package com.devlab.taskmaster.dto.report;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ReportDTO {

    private Integer id;

    private String generatedAt;

    private Integer totalTasks;

    private Integer tasksInBacklog;

    private Integer tasksInProgress;

    private Integer tasksCompleted;
}

File: TaskMasterWeb/TaskMasterWeb.Client/Dtos/Report/ReportDto.cs
namespace TaskMasterWeb.Client.Dtos.Report;

public class ReportDto
{
    public int Id { get; set; }

    public string GeneratedAt { get; set; } = default!;

    public int TotalTasks { get; set; }

    public int TasksInBacklog { get; set; }

    public int TasksInProgress { get; set; }

    public int TasksCompleted { get; set; }
}